### DISCUSSION TITLE
hadm_id for emergency patient transfered to ICU

### DISCUSSION THREAD

**Discussion by Jeanselme** [MIT-LCP/mimic-code#1274] [MIMIC-IV-ED] [answered]
Would a patient transfered to ICU have an hadm_id in the edstays table ?

**Comment by briangow** [comment, marked as answer]
Yes, a patient transferred from the Emergency Department to the ICU will have a `hadm_id` in `edstays`.

**Reply by Jeanselme** [reply]
Thank you so much !
